src/main/java/nextstep/ladder/domain/generator/Point.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.ladder.domain.generator;
+
+public class Point {
+    private final boolean current;
+    private final boolean next;
+
+    public Point(boolean current, boolean next) {
+        this.current = current;
+        this.next = next;
+    }
+
+    public boolean isConsecutive() {
+        return current && next;
+    }
+
+    public boolean current() {
+        return current;
+    }
+
+    public boolean next() {
+        return next;
+    }
```

Comment:
```suggestion
    public static Point first(boolean current) {
        return new Point(false, current);
    }
    
    public Point next(boolean current) {
        return new Point(this.current, current);
    }   
```

next가 아니라 previous 값을 가지는 구조로 구현하는 것은 어떨까?
그럴 경우 위와 같이 구현할 수 있지 않을까?